### PR TITLE
feat: Add label component

### PR DESCRIPTION
**Pull request by KonTra7cf**

### CODE REVIEW

File: src/components/Label/Label.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@import '../../../styles/mixins';
```
Comment (Veryyapeee) [inline src/components/Label/Label.scss:1]: You have to import this file to styles.scss where all of the styles are connected

File: src/components/Label/Label.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    padding: 25px 80px;
+    background-color: $btn-color;
+    color: $font-primary-color;
+    box-shadow: $box-shadow-btn-cateogry;
+	border-radius: 10px;
+	font-size: 32px;
+	font-family: "Inter";
+	text-transform: uppercase;
+	text-shadow: $box-shadow-btn-cateogry;
+	font-weight: 500;
```
Comment (Veryyapeee) [inline src/components/Label/Label.scss]: This is default font-weight I think

File: src/components/Label/Label.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    padding: 25px 80px;
+    background-color: $btn-color;
+    color: $font-primary-color;
+    box-shadow: $box-shadow-btn-cateogry;
+	border-radius: 10px;
+	font-size: 32px;
+	font-family: "Inter";
```
Comment (Veryyapeee) [inline src/components/Label/Label.scss]: Isn't it the default font in project?

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/Label/Label.scss (hunk 1 above)
Comment (KonTra7cf) [reply src/components/Label/Label.scss:1]: File imported to styles.scss

**Review by maciejopalinski — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**